apps/api/src/modules/server-api-keys/server-api-keys.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Resolver, Query, Args } from '@nestjs/graphql';
+import { ServerApiKeysService } from './server-api-keys.service';
+import { ServerApiKey } from './entities/server-api-key.entity';
+
+@Resolver(() => ServerApiKey)
+export class ServerApiKeysResolver {
```

Comment:
We shouldn't have an unprotected resolver for this. I also think we do not need this for the time being. If possible, we can remove this resolver entirely for now (along with the spec file) and add it later when we need.

Reply:
Removed this